### PR TITLE
Use constant for supported formats

### PR DESCRIPTION
**Pull request by J0WI**
Supported since [PHP 5.6](https://www.php.net/manual/en/migration56.new-features.php).

### CODE REVIEW

File: apps/encryption/lib/Crypto/Crypt.php [hunk 1]
```diff
@@ -57,10 +57,20 @@
  * @package OCA\Encryption\Crypto
  */
 class Crypt {
+	public const SUPPORTED_CIPHERS_AND_KEY_SIZE = [
```
Comment (rullzer) [inline apps/encryption/lib/Crypto/Crypt.php:60]: why does their visibility change all of a sudden?

Comment (J0WI) [reply apps/encryption/lib/Crypto/Crypt.php:60]: `DEFAULT_CIPHER` and `LEGACY_CIPHER` are also public, so IMHO it makes sense to have the same visibility for all available ciphers.

Comment (J0WI) [reply apps/encryption/lib/Crypto/Crypt.php:60]: @rullzer should I revert this change?

Comment (J0WI) [reply apps/encryption/lib/Crypto/Crypt.php:60]: #26383 depends on this

**Review by LukasReschke — APPROVED** [review]

**Comment by J0WI** [comment]
_asking @nextcloud/encryption for a second review :smiley:_

**Review by blizzz — APPROVED** [review]

**Pull request merged**